content/en/docs/kubernetes/getting-started.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Getting Started
+weight: 1
+# prettier-ignore
+cSpell:ignore: kubeletstats kubelet filelog sattributes sattributesprocessor kubeletstatsreceiver filelogreceiver loggingexporter otlpexporter sclusterreceiver sobjectsreceiver
+---
+
+This section will walk you through the fasted way to get started monitoring your
+Kubernetes cluster using OpenTelemetry. It will focus on collecting metrics and
+logs for Kubernetes clusters, nodes, pods, and containers as well as enabling
+the cluster to support services emitting OTLP data.
+
+If you're looking to see OpenTelemetry in action in Kubernetes the best place to
+start is the [OpenTelemetry Demo](../../demo/kubernetes-deployment). The demo is
+intended to illustrate the implementation of OpenTelemetry, but it is not
+intended to be an example of how to monitor Kubernetes itself. Once you're done
+with this walkthrough it can be a fun experiment to install the demo and see how
+all all the monitoring responds to an active workload.
```

Comment:
```suggestion
intended to be an example of how to monitor Kubernetes itself. Once you finish
with this walkthrough, it can be a fun experiment to install the demo and see how
all the monitoring responds to an active workload.
```